src/Column/MoneyColumn.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * Symfony DataTables Bundle
+ * (c) Omines Internetbureau B.V. - https://omines.nl/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle\Column;
+
+use Omines\DataTablesBundle\DataTable;
+use Omines\DataTablesBundle\Exception\MissingDependencyException;
+use Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * MoneyColumn.
+ *
+ * @author Max Mishyn <max.mishyn@wisetiger.co.uk>
+ */
+class MoneyColumn extends AbstractColumn
+{
+    /**
+     * @var MoneyToLocalizedStringTransformer
+     */
+    private $transformer;
+
+    public function normalize($value): string
+    {
+        return $this->isRaw() ? (string) $value : $this->format(intval($value));
+    }
+
+    protected function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $resolver
+            ->setDefaults([
+                'raw' => false,
+                'currency' => '',
+                'divisor' => 1,
+                'scale' => 2,
+            ])
+            ->setAllowedTypes('raw', 'bool')
+            ->setAllowedTypes('currency', 'string')
+            ->setAllowedTypes('divisor', 'int')
+            ->setAllowedTypes('scale', 'int')
+        ;
+
+        return $this;
+    }
+
+    public function initialize(string $name, int $index, array $options, DataTable $dataTable)
+    {
+        parent::initialize($name, $index, $options, $dataTable);
+        if (class_exists('Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer')) {
+            $this->transformer = new MoneyToLocalizedStringTransformer($this->options['scale'], true, \NumberFormatter::ROUND_HALFUP, $this->options['divisor']);
+        } else {
+            throw new MissingDependencyException('You must have Symfony\Form installed to use ' . self::class);
```

Comment:
Why depend on symfony/form instead of just the NumberFormatter?

Reply:
MoneyColumn accepts number in different storage formats, i.e. it can receive int value of 530 to represent "$5.30". So we have parameters to define divider (100 in the example above) and scale to define fixed decimal points. There code which does absolutely the same task in MoneyToLocalizedStringTransformer. My idea was that most probably in 99% of apps with DataTablesBundle there is Symfony's Forms bundle installed. So I decided to reuse it instead of duplicating it's code. If you think that this transforer's code should be in DataTables, I can move it to make bundle more self contained. Pls asvise.

Reply:
P.S.  NumberFormatter does not have this divider ability which is used quite often to store money values.

Reply:
Currently the bundle neither depends on nor suggests `symfony/form`, nor do I think it should purely for the reason of formatting a column value. We should prefer a full native implementation based purely on Intl extensions (if possible!) instead of borrowing code from unrelated packages and using regular expressions.

(admittedly, these are some of the reasons our own teams previously elected to implement MoneyColumn at the project level instead of the library level, as it allows more shortcuts like this).

Reply:
I'm absolutely fine with it. I'll implement transformer class inside of the lidrary. It is relatively simple class without any deeper dependencies.